src/datasources/transaction-api/transaction-api.service.ts

```diff
@@ -106,4 +107,31 @@ export class TransactionApi implements ITransactionApi {
       throw this.httpErrorFactory.from(error);
     }
   }
+
+  async getDelegates(
+    safeAddress?: string,
+    delegate?: string,
+    delegator?: string,
+    label?: string,
+    limit?: number,
+    offset?: number,
+  ): Promise<Page<Delegate>> {
+    try {
+      const cacheKey = `${this.chainId}_delegates`;
+      const cacheKeyField = `${safeAddress}_${delegate}_${delegator}_${label}`;
```

Comment:
Should we include both limit and offset as part of cache key? I mean, otherwise we would be caching different offset/limit results in the same key 🤔

Reply:
yes, updated